src/components/Contact.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 const Contact = () => {
   return (
-    <div className='w-full h-screen bg-gradient-to-b from-black to-gray-800 p-4 text-white'>
+    <div name="contact" className='w-full h-screen bg-gradient-to-b from-black to-gray-800 p-4 text-white pt-28'>
```

Comment:
name field was missing from contact.js component.
It is necessary for `react-scroll` module to work properly.